src/test/java/hw2/ex1/webDriver/DriverManager.java

```diff
@@ -0,0 +1,20 @@
+package hw2.ex1.webDriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.util.concurrent.TimeUnit;
+
+public class DriverManager {
+    private void setupChromeDriver() {
+        WebDriverManager.chromedriver().setup();
```

Comment:
This looks like one-time action and can be performed in the constructor (or in static-initialization and `setupDriver()` can become static)